### PR TITLE
Docker: Pin "ubuntu-latest" to 18.04

### PR DESCRIPTION
**Pull request by Jorteck**
~Required for PR: https://github.com/nwnxee/unified/pull/1200~
Required to stop everything breaking.

### CODE REVIEW

**Comment by Daztek** [comment]
What's ubuntu-latest at? 18.04? If so this would up the requirement for our downloadable binaries to 20.04 too I think

**Comment by Jorteck** [comment]
It's currently being rolled out to 20.04: https://github.com/actions/virtual-environments/issues/1816

**Comment by Daztek** [comment]
Do you know if stuff compiled with 20.04 still works in beamdog/nwserver's debian buster?

**Comment by Jorteck** [comment]
Doesn't look like it - `libstdc++6` is one version off (Ubuntu: 3.4.26/6.0.26, Debian: 3.4.25/6.0.25):

> ./nwserver-linux: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version 'GLIBCXX_3.4.26' not found (required by /home/debian/nwn-server/bin/linux-x86/NWNX_Core.so)

https://packages.debian.org/buster/amd64/libstdc++6/filelist

**Comment by Daztek** [comment]
Don't think we can switch to 20.04 in that case and should probably pin it to 18.04 before everything blows up.

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```
Comment (Daztek) [inline .github/workflows/docs.yml:21]: I think docs has to be on 20.04 for doxygen

Comment (plenarius) [reply .github/workflows/docs.yml:21]: Yes, docs needs to be on 20.04 please, we need doxygen >= 1.8.15

**Comment by Daztek** [comment]
If this breaks everything, it wasn't me

**Pull request merged**